templates/cook/cook_detail.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% block content %}
+
+  <div class="container my-5">
+    <div class="card">
+      <img src="https://qudahalloween.com/cdn/shop/articles/Crazy-Dave-Costume_600x.png?v=1719396046"
```

Comment:
add width and height properties, or at least max-width

Reply:
fixed.